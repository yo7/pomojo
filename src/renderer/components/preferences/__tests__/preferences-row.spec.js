import {shallow} from 'vue-test-utils'
import {createRenderer} from 'vue-server-renderer'
import PreferencesRow from '../preferences-row.vue'

describe('PreferencesRow', () => {
  const wrapper = shallow(PreferencesRow, {propsData: {
    title: 'Title'
  }})

  it('renders title', () => {
    expect(wrapper.find('.title').text()).toBe('Title')
  })

  it('renders correctly', () => {
    const renderer = createRenderer()
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) {
        throw new Error()
      }
      expect(str).toMatchSnapshot()
    })
  })
})
