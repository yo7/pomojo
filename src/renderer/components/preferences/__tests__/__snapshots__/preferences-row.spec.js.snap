// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreferencesRow renders correctly 1`] = `
<div class="preferences-row">
    <p class="title">Title</p>
</div>
`;
